rocAL/include/decoders/image/decoder.h

```diff
@@ -32,12 +32,12 @@ enum class DecoderType {
     TURBO_JPEG = 0,        //!< Can only decode
     FUSED_TURBO_JPEG = 1,  //!< FOR PARTIAL DECODING
     OPENCV_DEC = 2,        //!< for back_up decoding
-    HW_JPEG_DEC = 3,
+    HW_JPEG_DEC = 3,       //!< for JPEG decoding using HW via FFMPEG  
     SKIP_DECODE = 4,       //!< For skipping decoding in case of uncompressed data from reader
-    OVX_FFMPEG = 5,        //!< Uses FFMPEG to decode video streams, can decode up to 4 video streams simultaneously
-    FFMPEG_SOFTWARE_DECODE = 6,
-    FFMPEG_HARDWARE_DECODE = 7,
-    AUDIO_SOFTWARE_DECODE = 8   //!< Uses sndfile to decode audio files
+    FFMPEG_SW_DECODE = 6,   //!< for video decoding using CPU and FFMPEG
```

Comment:
Why is 5 not being used?. If we remove OVX_FFMPWG, please make FFMPEG_SW_DECODE=5